contracts/examples/guards/DebugTransactionGuard.sol

```diff
@@ -24,13 +26,29 @@ contract DebugTransactionGuard is BaseGuard {
         Enum.Operation operation,
         uint256 safeTxGas,
         bool usesRefund,
-        uint256 nonce
+        uint256 nonce,
+        bytes signatures,
+        address executor
```

Comment:
I've added more debug values here, felt like they were undeservedly omitted. Let me know if I should revert.